api/rest/client/lbclient.go

```diff
@@ -369,6 +369,19 @@ func (lc *loadBalancingClient) Metrics(ctx context.Context, name string) ([]*api
 	return metrics, err
 }
 
+// MetricNames returns the list of metrics.
+func (lc *loadBalancingClient) MetricNames(ctx context.Context) ([]string, error) {
```

Comment:
The loadbalancing client should have tracing as well. Though that may be a seperate PR since all the methods are missing it.

Reply:
Yes